back/src/chat/channel/channel.service.ts

```diff
@@ -150,9 +171,10 @@ export class ChannelService {
 				});
 			case ChannelRole.INVITED:
 				channelConnection = await this.updateChannelRole(ChannelRole.DEFAULT, targetChannelId, user.id);
+				this.chatService.emit('chat:join', channelConnection, targetChannelId.toString());
+				this.chatService.joinRoom(socketId, targetChannelId.toString());
 		}
-
-		// TODO: Join the socket.id, if specified, to the socket.io room
+		this.chatService.emit('chat:join', channelConnection, user.id.toString());
```

Comment:
```suggestion
		this.chatService.emit('chat:join', channelConnection, socketId);
```

Reply:
I made some changes.
I'm still unsure if I should notify the user who is trying to join via websockets that he indeed joined as he already gets the response to the request...